tests/Unit/ProtoTests/Basic/out/src/Client/BasicClient.php

```diff
@@ -31,6 +31,8 @@
 use Google\ApiCore\Transport\TransportInterface;
 use Google\ApiCore\ValidationException;
 use Google\Auth\FetchAuthTokenInterface;
+use Google\Cloud\Core\InsecureCredentialsWrapper;
```

Comment:
This is available only in the Core. @bshaffer WDYT? Should we move this class to gax?

Reply:
Yes, I think that's a good idea. We can then deprecate this one, and put the new one in the `Google\ApiCore` namespace

Reply:
Updated this, gax needs new release for this though

Reply:
This has been done in https://github.com/googleapis/google-cloud-php/pull/7089!